Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocListControlObjectTest.cs

```diff
@@ -867,7 +871,11 @@ public void TestGetCellMatchesExactly ()
       // Set Timeout to Zero so we don't have to wait the full timeout for the exception
       bocList.Scope.ElementFinder.Options.Timeout = TimeSpan.Zero;
 
-      Assert.That (() => bocList.GetCellWhere ("Title", "EO"), Throws.InstanceOf<MissingHtmlException>());
+      Assert.That (
+          () => bocList.GetCellWhere ("Title", "EO"),
+          Throws.InstanceOf<WebTestException>()
+              .With.Message.EqualTo ("The element cannot be found.")
+              .And.InnerException.InstanceOf<MissingHtmlException>());

```

Comment:
Because here the Inner Exception is relevant, as it contains information (selector) about the element that cannot be found. In the other tests (as far as i have seen), the exception itself does not contain other information that we need